File: Easy/Python/1374. Generate a String With Characters That Have Odd Counts.py
class Solution(object):
    def generateTheString(self, n):
        odd = ''

        for i in range(n - 1):
            odd = odd + 'a'
        
        if n % 2 == 0:
            odd = odd + 'b'
        else:
            odd = odd + 'a'
        
        return odd

File: Easy/Java/1374. Generate a String With Characters That Have Odd Counts.java
class Solution {
    public String generateTheString(int n) {
        String odd = "";

        for(int i = 0; i < n - 1; i++){
            odd += "a";
        }

        if(n % 2 == 0)
            odd += "b";
        else
            odd += "a";

        return odd;
    }
}
